utils/src/main/scala/it/pagopa/interop/commons/logging/logging.scala

```diff
@@ -21,19 +21,19 @@ package object logging {
     if (config.hasPath("interop-commons.isInternetFacing")) config.getBoolean("interop-commons.isInternetFacing")
     else false
 
-  private def contexts(values: Map[String, String]): String = {
+  private def createLogContexts(values: Map[String, String]): String = {
     val ipAddress: String     = values.getOrElse(IP_ADDRESS, "")
-    val subject: String       = values.get(UID).filterNot(_.isBlank).orElse(values.get(SUB)).getOrElse("")
+    val uid: String           = values.get(UID).filterNot(_.isBlank).orElse(values.get(SUB)).getOrElse("")
     val correlationId: String = values.getOrElse(CORRELATION_ID_HEADER, "")
-    s"[$ipAddress] [$subject] [$correlationId]"
+    s"[IP=$ipAddress] [UID=$uid] [CORR_ID=$correlationId]"
   }
 
   /** Defines log message decoration for Interop
     */
   implicit case object CanLogContextFields extends CanLog[ContextFieldsToLog] {
     override def logMessage(originalMsg: String, fields: ContextFieldsToLog): String = {
       val fieldsMap: Map[String, String] = fields.toMap
-      s"[${fieldsMap.getOrElse(UID, "")}] [${fieldsMap.getOrElse(CORRELATION_ID_HEADER, "")}] - $originalMsg"
+      s"${createLogContexts(fieldsMap)} - $originalMsg"
```

Comment:
As you can see, I applied the same pattern here.
Do you see something wrong here?